ruff_cli/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "ruff_cli"
+version = "0.0.219"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ruff = { path = ".." }
+
+annotate-snippets = { version = "0.9.1", features = ["color"] }
+anyhow = { version = "1.0.66" }
+atty = { version = "0.2.14" }
+bincode = { version = "1.3.3" }
+cachedir = { version = "0.3.0" }
+chrono = { version = "0.4.21", default-features = false, features = ["clock"] }
+clap = { version = "4.0.1", features = ["derive", "env"] }
+clap_complete_command = { version = "0.4.0" }
+clearscreen = { version = "2.0.0" }
+colored = { version = "2.0.0" }
+filetime = { version = "0.2.17" }
+glob = { version = "0.3.0" }
+ignore = { version = "0.4.18" }
+itertools = { version = "0.10.5" }
+log = { version = "0.4.17" }
+notify = { version = "5.0.0" }
+path-absolutize = { version = "3.0.14", features = ["once_cell_cache"] }
+quick-junit = { version = "0.3.2" }
+rayon = { version = "1.5.3" }
+regex = { version = "1.6.0" }
+rustc-hash = { version = "1.1.0" }
+rustpython-ast = { features = ["unparse"], git = "https://github.com/RustPython/RustPython.git", rev = "d532160333ffeb6dbeca2c2728c2391cd1e53b7f" }
```

Comment:
You can use cargo workspace inheritance feature to manage common dependencies between workspace members, see https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table

Reply:
Oh neat, I didn't know about that!

I guess you meant specifically for these verbose dependencies on `rustpython-*` right? I just changed `ruff::message` to `pub use rustpython_ast::Location;` and dropped the direct dependencies on rustpython-* from `ruff_cli`, which I think is even better :)